src/components/Dialog/ModelessDialog.tsx

```diff
@@ -76,8 +78,16 @@ type Props = {
    * ポータルの container となる DOM 要素を追加する親要素
    */
   portalParent?: HTMLElement | RefObject<HTMLElement>
+  /** コンポーネント内の文言を変更するための関数を設定 */
+  decorators?: DecoratorsType<'closeButtonIconAlt'> & {
+    dialogHandlerAriaLabel?: (txt: string) => string
+    dialogHandlerAriaValuetext?: (txt: string, data: DOMRect | undefined) => string
```

Comment:
`undefined` って明示する必要がありそうですか?

```suggestion
    dialogHandlerAriaValuetext?: (txt: string, data?: DOMRect) => string
```

Reply:
🤔 
この関数には常にdataが設定されており、その値は　DOMRect | undefined なので 上の方が正確なはず...
`data?: DOMRect` だと `data は 渡されないか DOMRect | undefined` とかの判定になる気が.

Reply:
意図理解しました!
パッと見で意図が読めるほどの違いはないなとは思いました。`data` 渡されないのも結局 `DOMRect | undefined` にはなるので。

Reply:
@uknmr 
TSの設定によっては `data?: DOMRect` だとエラーになったりするので厳密なまま行きますねー